tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "lib": ["ES2017", "dom"],
+    "lib": ["es2017", "dom"],
```

Comment:
1. should avoid code change that is not related to commit message
1. why lower case? doc uses upper case too: https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
i receive this message from vscode and I thought it may be a hint of incorrect value
```
Value is not accepted. Valid values: "es5", "es6", "es2015", "es7", "es2016", "es2017", "es2018", "esnext", "dom", "dom.iterable", "webworker", "scripthost", "es2015.core", "es2015.collection", "es2015.generator", "es2015.iterable", "es2015.promise", "es2015.proxy", "es2015.reflect", "es2015.symbol", "es2015.symbol.wellknown", "es2016.array.include", "es2017.object", "es2017.intl", "es2017.sharedmemory", "es2017.string", "es2017.typedarrays", "es2018.intl", "es2018.promise", "es2018.regexp", "esnext.asynciterable", "esnext.array", "esnext.intl", "esnext.symbol".
```

But I should make changes that relate to the PR. Thanks